tts/Time.def.lua

```diff
@@ -0,0 +1,6 @@
+---@class Time
+---@field time number
+
+local Time
```

Comment:
Did you mean to commit this file? The above won't work because `Time` is a local and due to the new line between the doc and the variable declaration.

Reply:
I did. But I'll grant the mistake.  I've corrected, but only with a 50% confidence in the correctness of my correction.